BlueprintUI/Tests/UIAccessibililtyTraitsTests.swift

```diff
@@ -0,0 +1,45 @@
+import BlueprintUI
+import UIKit
+import XCTest
+
+
+class UIAccessibilityTraits_Tests: XCTestCase {
+
+    func test_backButton() throws {
+        let root = UIViewController()
+        let second = UIViewController()
+        let navigationController = UINavigationController()
+        navigationController.viewControllers = [root, second]
+
+        try show(vc: navigationController) { _ in
+            let button = try XCTUnwrap(
+                navigationController.navigationBar.findSubview(ofType: UIControl.self)
+            )
+
+            /// The `.backButton` trait is private, so this test is here to verify it does not change.
+
+            XCTAssertEqual(button.accessibilityTraits, .backButton)
+        }
+    }
+
+    func test_toggleSwitch() throws {
+
+        if #available(iOS 17.0, *) {
+            /// The `.toggleButton` trait is exposed iOS 17, so this test is here to verify it matches ours.
+            XCTAssertEqual(UIAccessibilityTraits._toggleButton, UIAccessibilityTraits.toggleButton)
+        }
+
+        guard let rootVC = UIApplication.shared.delegate?.window??.rootViewController else {
+            XCTFail("No root vc, Shared application:\(UIApplication.shared) - Delegate: \(String(describing: UIApplication.shared.delegate))")
+            return
+        }
+        let uiSwitch = UISwitch()
+        rootVC.view.addSubview(uiSwitch)
+        uiSwitch.layoutIfNeeded()
+
+        let recursiveDescription = uiSwitch.window?.perform(Selector(("recursiveDescription"))).takeUnretainedValue()
+        add(XCTAttachment(string: String(describing: recursiveDescription)))
```

Comment:
this attached view hierarchy matches that from my local machine, confirming that the switch is within the view hierarchy of the window.  <img width="1385" alt="Screenshot 2025-02-20 at 13 59 30" src="https://github.com/user-attachments/assets/991b7430-c9de-404a-8a09-79981d590dd8" />